README.md

```diff
@@ -32,12 +32,12 @@ As well as the following dynamic check:
 
 Note that `provenance` and the dynamic checks require the prescence of build metadata in your metadata store. While unsigned metadata is valid, to ensure that you are trusting metadata that hasn't been forged, it is recommended that you use signed metadata as well.
 
-## Voucher Server and Client
+## Voucher Server, Subscriber, and Client
 
-This repository contains two tools: Voucher server, intended to run in your infrastructure to respond to CI/CD pipeline requests, and Voucher client, an example of a Voucher API client that you can use directly in your CI/CD pipeline or as a basis for your own code.
-
-- [Server](cmd/voucher_server/README.md)
-- [Client](cmd/voucher_client/README.md)
+This repository contains three tools:
+- [Voucher server](cmd/voucher_server/README.md): intended to run in your infrastructure to respond to CI/CD pipeline requests.
+- [Voucher subscriber](cmd/voucher_subscriber/README.md): intended to listen for pub/sub messages for image creations and automatically vouch images.
+- [Voucher client](cmd/voucher_client/README.md): an example of a Voucher API client that you can use directly in your CI/CD pipeline or as a basis for your own code.
```

Comment:
Maybe worth mentioning that the client sends requests to the Voucher Server, since there are three entities now.